Automattic/woocommerce-payments
# Add usage notice to platform checkout setting

#### Changes proposed in this Pull Request

This PR adds the required notice to the platform checkout settings page, letting merchants know that by using platform checkout they agree to our Terms of Service and Privacy Policy.

Additional Context: pdpOw2-rE-p2, pdpOw2-l8-p2

#### Testing instructions

- Visit WCPay settings, ensure that you see the notice under the platform checkout option.
- Check all the links to verify that they work.

cc @malithsen @c-shultz 

-------------------


**Post merge**
